### PR TITLE
Format and remove `assert`

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -96,32 +96,45 @@ main(void) {
                                                           .get_identity           = get_identity_cb,
                                                           .get_user_provided_keys = NULL };
 
-    assert(MENDER_OK == mender_client_init(&mender_client_config, &mender_client_callbacks));
+    if (MENDER_OK != mender_client_init(&mender_client_config, &mender_client_callbacks)) {
+        LOG_INF("Failed to initialize the client");
```
Comment (danielskinstad) [inline src/main.c:100]: Why not `LOG_ERR`?

Comment (vpodzime) [reply src/main.c:100]: yup!

**Review by vpodzime — COMMENTED** [review]
Looks good to me otherwise.

File: src/main.c (hunk 1 above)
Comment (lluiscampos) [reply src/main.c:100]: :see_no_evil:

**Review by danielskinstad — APPROVED** [review]

**Pull request merged**